ts/features/bonus/siciliaVola/screens/voucherList/VoucherListScreen.tsx

```diff
@@ -0,0 +1,204 @@
+
+  const vouchers = toArray(props.indexedVouchers);
+
+  useEffect(() => {
+    props.requestVoucherState();
+    props.resetFilter();
+  }, []);
+
+  useEffect(() => {
+    if (isUndefined(props.requiredDataLoaded)) {
+      props.requestVoucherPage(props.filters);
+    }
+  }, [props.filters, props.requiredDataLoaded]);
+
+  const openFiltersModal = () =>
+    showAnimatedModal(
+      <SvVoucherListFilters onClose={hideModal} onConfirm={hideModal} />,
+      BottomTopAnimation
+    );
+
+  return (
+    <BaseScreenComponent
+      goBack={true}
+      contextualHelp={emptyContextualHelp}
+      headerTitle={I18n.t("bonus.sv.headerTitle")}
+      isSearchAvailable={{ enabled: true, onSearchTap: openFiltersModal }}
+    >
+      <SafeAreaView style={IOStyles.flex} testID={"VoucherListScreen"}>
+        {!isReady(props.possibleVoucherState) ? (
+          <LoadingErrorComponent
+            isLoading={isLoading(props.possibleVoucherState)}
+            loadingCaption={I18n.t("bonus.sv.voucherList.loading")}
+            onRetry={props.requestVoucherState}
+          />
+        ) : (
+          <>
+            <H1 style={IOStyles.horizontalContentPadding}>
+              {I18n.t("bonus.sv.voucherList.title")}
+            </H1>
+            <View spacer />
+
+            {isReady(props.requiredDataLoaded) && vouchers.length === 0 ? (
+              <EmptyVoucherList />
+            ) : (
+              <FlatList
+                style={[IOStyles.horizontalContentPadding]}
+                data={vouchers}
+                ListFooterComponent={
+                  isLoading(props.requiredDataLoaded) && <FooterLoading />
+                }
+                keyExtractor={v => v.idVoucher.toString()}
+                ItemSeparatorComponent={() => (
+                  <ItemSeparatorComponent noPadded={true} />
+                )}
+                onEndReached={() => {
+                  if (props.uiParameters.nextPage !== undefined) {
+                    props.requestVoucherPage(props.filters);
+                  }
+                }}
+                onEndReachedThreshold={0.5}
+                renderItem={v => (
+                  <RenderItem
+                    idVoucher={v.item.idVoucher}
+                    departureDate={v.item.departureDate}
+                    destination={v.item.destination}
+                  />
+                )}
+                scrollEnabled={true}
+                keyboardShouldPersistTaps={"handled"}
+              />
+            )}
+          </>
+        )}
+      </SafeAreaView>
+    </BaseScreenComponent>
+  );
+};
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  requestVoucherState: () => dispatch(svPossibleVoucherStateGet.request()),
+  resetFilter: () => dispatch(svResetFilter({})),
+  requestVoucherPage: (filters: FilterState) =>
+    dispatch(svVoucherListGet.request(filters)),
+  start: () => dispatch(svGenerateVoucherStart())
```

Comment:
```suggestion
```
This is not used anymore.

Reply:
Removed in [95f3aa3](https://github.com/pagopa/io-app/commit/95f3aa3c8e47c3089281ee6c807cee854cbfaf17)